specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-04-01-preview/authorization-RoleAssignmentPaginationCalls.json

```diff
@@ -0,0 +1,194 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AuthorizationManagementClient",
+    "version": "2021-04-01-preview",
+    "description": "Role based access control provides you a way to apply granular level policy administration down to individual resources or resource groups. These operations enable you to manage role assignments. A role assignment grants access to Azure Active Directory users."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Authorization/roleAssignments": {
+      "get": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_ListForProvider",
```

Comment:
operation ID (RoleAssignments_ListForProvider) is ambiguous.
who's the provider in this case? if it's Microsoft.Auth please make it more explicit  
If the provider is part of the scope please make it more explicit wwith something like this  {scope}/provider/{resourceProvider}/privider/Micropsoft.Auth[...]

Reply:
Updated the operationId and description